src/InstrumentationEngine.Lib/LoggerService.cpp

```diff
@@ -73,6 +74,23 @@ HRESULT CLoggerService::GetLoggingFlags(_Out_ LoggingFlags* pLoggingFlags)
     return S_OK;
 }
 
+HRESULT CLoggerService::GetInstruMethodLoggingFlags(_Out_ LoggingFlags* pLoggingFlags)
```

Comment:
>GetInstruMethodLoggingFlags [](start = 24, length = 27)

It's a bit strange for ProfilerManager to query this each time before setting it again (and we lose track of which method is logging at which level). I'm revamping this API so instead of Get/Set, there will only be one UpdateInstrumentationMethodLoggingFlag that updates a dictionary in CLoggerService